doc/user-how-to/troubleshooting.md

```diff
@@ -1,6 +1,62 @@
 # Troubleshooting
 
-## "Too many open files" on Linux
+```{contents} Contents
```

Comment:
Maybe instead of using a `.. contents::` directive, we should break all these items into their own files in a `troubleshooting` subdirectory with an `index.rst` and `.. toctree::`? :shrug: I think this is fine for now?